### PR TITLE
variable changes

### PR DESCRIPTION
**Pull request by ashish-um**
- canvas size changed, cuz too much space was empty.
- a new particle radius variable introduced.
- some other variable value changes to make it look more natural.

### CODE REVIEW

File: simulation.js [hunk 1]
```diff
@@ -2,16 +2,17 @@ class Simulation {
 	constructor() {
 		this.particles = [];
 
-		this.AMOUNT_PARTICLES = 2000;
+		this.AMOUNT_PARTICLES = 1200;
 		this.VELOCITY_DAMPING = 1;
```
Comment (DarkMatter-999) [inline simulation.js:9]: is 2000 particles too high for a meaningful fps?

File: index.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
 				background: #242321;
 			}
 		</style>
-		<canvas id="canvas" width="1280px" height="720px" style="border: solid 1px black"></canvas>
+		<canvas id="canvas" width="1080px" height="720px" style="border: solid 1px black"></canvas>
```
Comment (DarkMatter-999) [inline index.html:20]: I think you wanted to change the height to 1080 and width to 1920

earlier it was 1280 now its 1080 making it "smaller"

**Comment by DarkMatter-999** [comment]
Please look into the review and i guess lesser particles is okay, i wanted to see the limits of the simulation, but the change in the resolution for making is bigger is incorrect

**Review by DarkMatter-999 — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (ashish-um) [reply index.html:20]: yeah i wanted to make it smaller, cuz ones the particles settle, there's too much empty space visible

File: simulation.js (hunk 1 above)
Comment (ashish-um) [reply simulation.js:9]: for my laptop, it was when on battery power

File: index.html (hunk 2 above)
Comment (DarkMatter-999) [reply index.html:20]: Now that I have added particle emitters so that will be a nonissue cause we have infinite particles

Comment (ashish-um) [reply index.html:20]: yes that'll do it

**Review by ashish-um — COMMENTED** [review]
ok cool

File: simulation.js (hunk 1 above)
Comment (DarkMatter-999) [reply simulation.js:9]: This wont be a problem now that we have particle emitters.

**Pull request closed without merging**